nemo/collections/asr/models/label_models.py

```diff
@@ -373,14 +373,17 @@ def evaluation_step(self, batch, batch_idx, dataloader_idx: int = 0, tag: str =
         self._macro_accuracy.update(preds=logits, target=labels)
         stats = self._macro_accuracy._final_state()
 
-        return {
+        eval_dict = {
             f'{tag}_loss': loss_value,
             f'{tag}_correct_counts': correct_counts,
             f'{tag}_total_counts': total_counts,
             f'{tag}_acc_micro_top_k': acc_top_k,
             f'{tag}_acc_macro_stats': stats,
         }
 
+        self.log_dict(eval_dict)
```

Comment:
not all variables in `eval_dict` need logging, please move `self.log()` into `multi_evaluation_epoch_end` where it calculates the averaged metrics

Reply:
I think all these variables were logged previously too (example run - https://wandb.ai/nvidia/titanet-chime7-training?workspace=user-kdhawan), please let me know if you want me to remove some of these from the log